config/overlays/openshift/role_binding_cluster_monitoring_view.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can you add a comment to explain why the operator needs this privilege (so it can grant it to the individual Tempo CR Service Accounts)?

Reply:
done